### PR TITLE
add-structure-proyect

### PR DESCRIPTION
**Pull request by Jazminmazzu**

### CODE REVIEW

File: src/components/cryptoList/styles.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import styled from 'styled-components';
+import theme from '../../config/utils/theme';
+import { Text as TextRN, Image as ImageRN, View as ViewRN } from 'react-native';
```
Comment (VicenBayugar) [inline src/components/cryptoList/styles.ts:3]: why do you rename RN components?

File: src/components/list/index.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  ContainerChange,
+  Arrow,
+  TextPercent,
+  TextPercentage,
+  Line,
+} from '../list/styles';
+import GreenArrow from '../../assets/img/GreenArrow.png';
+import RedArrow from '../../assets/img/RedArrow.png';
+
+const List = ({item}: {item: Crypto}) => (
```
Comment (tomasdematos) [inline src/components/list/index.tsx:21]: move the type to and interface::

interface Props {
   item: Crypto;
};

{item}: Props

File: src/components/list/index.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+          <TextSymbol>{item.symbol}</TextSymbol>
+        </ContainerText>
+      </ContainerLeft>
+      <ContainerRight>
+        <ContainerChange>
+          <TextCoin>${item.price}</TextCoin>
+          <TextPercent>
+            <Arrow source={item.percent > 0 ? GreenArrow : RedArrow} />
+            <TextPercentage positive={item.percent > 0}>
+              {item.percent}%{' '}
```
Comment (tomasdematos) [inline src/components/list/index.tsx:39]: delete {' '}

File: src/components/cryptoList/styles.ts (hunk 1 above)
Comment (Jazminmazzu) [reply src/components/cryptoList/styles.ts:3]: It is the alternative that I found because otherwise it returned an error.

Comment (VicenBayugar) [reply src/components/cryptoList/styles.ts:3]: Probably the error was that you were importing Text from RN and then you were creating a Text component with styled components (same name)

File: src/components/list/index.tsx [hunk 4]
```diff
@@ -2,45 +2,47 @@ import React from 'react';
 import {Crypto} from '../interfaces/index';
 import {
   Container,
-  ContainerLogo,
+  LogoContainer,
   CryptoLogo,
-  ContainerText,
+  TextContainer,
   TextCoin,
```
Comment (tomasdematos) [inline src/components/list/index.tsx:8]: Rename all with the correct format:

SomethingText, SomethingContainer, etc..

**Pull request merged**